public/app/features/alerting/unified/components/rule-editor/NotificationsStep.tsx

```diff
@@ -0,0 +1,60 @@
+import { css } from '@emotion/css';
+import React, { FC, useState } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Card, Link, useStyles2, useTheme2 } from '@grafana/ui';
+
+import { RuleEditorSection } from './RuleEditorSection';
+
+export const NotificationsStep: FC = () => {
+  const [hideFlowChart, setHideFlowChart] = useState(false);
+  const styles = useStyles2(getStyles);
+  const theme = useTheme2();
+
+  return (
+    <RuleEditorSection
+      stepNo={4}
+      title="Notifications"
+      description="Grafana handles the notifications for alerts by assigning labels to alerts. These labels connect alerts to contact points and silence alert instances that have matching labels."
```

Comment:
nit: I think this mixes some things together - labels are assigned to alerts, and then those labelled alerts go through several sets of rules to decide what gets sent to the contact points.
1. Where do I send these alerts?
2. Are the alerts grouped so only a single alert is sent?
3. Are there any silences or mute timings that stop these alerts from being sent?

These questions are answered by both notification policies and silences - the labels don't "do" anything.

Here's a suggestion, but feel free to stick with what's here already.
> Grafana assigns labels to alerts based on the results of the query and the configuration of the alerting rule. These labels are compared with notification rules and silences to select the correct contact point and determine if an alert has been silenced and should not be sent.

Reply:
@joeblubaugh it's a bit long for a description. I will merge as is and we can figure something out in beta.